Sources/NIOHTTP2/HTTP2Frame.swift

```diff
@@ -39,7 +184,7 @@ public struct HTTP2Frame {
         /// A DATA frame, containing raw bytes.
         ///
         /// See [RFC 7540 § 6.1](https://httpwg.org/specs/rfc7540.html#rfc.section.6.1).
-        case data(IOData)
+        case data(Data)
```

Comment:
I find this reads a bit confusing because (`Foundation.`)`Data` is such a well-known type but here you're referring to `FramePayload.Data`...

Reply:
Would it be helpful to use the explicit fully qualified type name, or do you want a new name here?

Reply:
up to you but I think qualifying is good enough

Reply:
Resolved.